pkg/ruler/ruler.go

```diff
@@ -551,14 +557,14 @@ func (r *Ruler) getLocalRules(userID string) ([]*rules.RuleGroupDesc, error) {
 			default:
 				return nil, errors.Errorf("failed to assert type of rule '%v'", rule.Name())
 			}
-			groupDesc.Rules = append(groupDesc.Rules, ruleDesc)
+			groupDesc.ActiveRules = append(groupDesc.ActiveRules, ruleDesc)
 		}
 		groupDescs = append(groupDescs, groupDesc)
 	}
 	return groupDescs, nil
 }
 
-func (r *Ruler) getShardedRules(ctx context.Context, userID string) ([]*rules.RuleGroupDesc, error) {
+func (r *Ruler) getShardedRules(ctx context.Context, userID string) ([]*GroupStateDesc, error) {
```

Comment:
userID parameter is unused here

Reply:
Makes sense, it should be safe to remove.